### DISCUSSION TITLE
ddd

### DISCUSSION THREAD

**Discussion by snuped** [wladradchenko/wunjo.wladradchenko.ru#42] [General]

**Comment by wladradchenko** [comment]
Hello. I have been train model from this Huggingface [repository](https://huggingface.co/wladradchenko/wunjo.wladradchenko.ru). If you will see other link what that means a model from open source repository with free license. 

I recorded a [video](https://www.youtube.com/watch?v=Ystz4sFsmwU) about neural network models which using for video, about repositories and how it works. In further time will be publicate video about sound models. However you will able to find links in README.
